Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -44,7 +44,8 @@ bool IsChannel(u64 title_id)
 {
   return IsTitleType(title_id, TitleType::Channel) ||
          IsTitleType(title_id, TitleType::SystemChannel) ||
-         IsTitleType(title_id, TitleType::GameWithChannel);
+         IsTitleType(title_id, TitleType::GameWithChannel) ||
+         IsTitleType(title_id, TitleType::HiddenChannel) || title_id == TITLEID_SYSMENU;
```

Comment:
Any thoughts on checking SYSMENU first? Mostly because I dislike it sticking around there in the back, while the others have their own line :P
Or does clang-format scream if you do that?